internal/interconnections/interconnections.go

```diff
@@ -0,0 +1,71 @@
+// Copyright © 2018 Jasmin Gacic <jasmin@stackpointcloud.com>
+//
+// Permission is hereby granted, free of charge, to any person obtaining a copy
+// of this software and associated documentation files (the "Software"), to deal
+// in the Software without restriction, including without limitation the rights
+// to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+// copies of the Software, and to permit persons to whom the Software is
+// furnished to do so, subject to the following conditions:
+//
+// The above copyright notice and this permission notice shall be included in
+// all copies or substantial portions of the Software.
+//
+// THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+// IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+// FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+// AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+// LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+// OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+// THE SOFTWARE.
+
+package interconnections
+
+import (
+	metal "github.com/equinix-labs/metal-go/metal/v1"
+	"github.com/equinix/metal-cli/internal/outputs"
+	"github.com/spf13/cobra"
+)
+
+type Client struct {
+	Servicer Servicer
+	Service  metal.InterconnectionsApiService
+	Out      outputs.Outputer
+}
+
+func (c *Client) NewCommand() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:     `interconnections`,
+		Aliases: []string{"conn"},
+		Short:   "interconnections operations: get.",
```

Comment:
Is `create` also supported?  I see a file for it

Reply:
Yeah looks like this is the text that gets used when you just do "metal help" and it's definitely missing the "create" command in that listing.